readme.txt

```diff
@@ -82,8 +82,9 @@ Yes. It will work perfectly with all post-types & taxonomies that define their o
 Yes. The autoloader can be filtered, so adding new object types is as simple as:
 
 `
-add_filter( 'wp_get_default_user_activity_types', function( $types = array() ) {
-	$types[] = 'Your_New_Type' // class that extends WP_User_Activity_Type
+add_filter( 'wp_get_default_user_activity_types', function ( $types ) {
```

Comment:
I think it is always an array https://github.com/stuttter/wp-user-activity/blob/686bc51cf6524968fe20b20cba9bbd4af8e78f89/wp-user-activity/includes/common.php#L31